server/common.js

```diff
@@ -1,5 +1,8 @@
-module.exports = {serverHost : "lihi-test-db.mysql.database.azure.com",
-                  serverUser : "lihi_test_admin@lihi-test-db",
+module.exports = {
+                //   serverHost : "lihi-test-db.mysql.database.azure.com",
+                //   serverUser : "lihi_test_admin@lihi-test-db",
+                  serverHost : "lihi-dev-db.mysql.database.azure.com",
+                  serverUser : "lihi_dev_admin@lihi-dev-db",

```

Comment:
Do we want the test db or dev db? master currently has the test db in commons.js

Reply:
I think we want the dev one when we merge it so that our database changes go through when we remake it